source/includes/config-repo/_021-parse-directory.md.erb

```diff
@@ -443,6 +444,8 @@ mandatory. Some noteworthy properties of this element are:
     - Most top-level attributes of a pipeline *can* have a `location` defined. If defined, that value will be used to
       show more meaningful error messages, if the GoCD server detects errors during its parsing and validation.
 
+    - The `display_order_weight` property will be used to decide the order in which the pipelines will be shown on the GoCD dashboard.
```

Comment:
Document default value?

Reply:
Makes sense. Pushed a commit: https://github.com/gocd/plugin-api.go.cd/commit/ef325d4707decfca9f9de73de3de45eb115a80ee

It adds: `If not provided, it defaults to -1`.